File: aspnet-core/src/ERP.Healthcare.Application.Contracts/Patients/PatientDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace ERP.Healthcare.Patients
{
    public class PatientDto : EntityDto<int>
    {
        public int DoctorId { get; set; }

        public string DoctorName { get; set; }

        public string Name { get; set; }

        public DateTime BirthDate { get; set; }

        public string ShortDescription { get; set; }
    }
}

File: angular/src/app/patient/models/patient-dto.ts


import { EntityDto } from '@abp/ng.core';

export class PatientDto extends EntityDto<number> {
  doctorId: number;
  doctorName: string;
  name: string;
  birthDate: string;
  shortDescription: string;
  id: number;

  constructor(initialValues: Partial<PatientDto> = {}) {
    super(initialValues);
  }
}
